### PR TITLE
DataChannel support

### PR DESCRIPTION
**Pull request by jbenet**
These commits add support for the WebRTC DataChannel API.

### CODE REVIEW

**Comment by jbenet** [comment]
See https://github.com/webRTC/webrtc.io-client/pull/24

File: example/index.html [hunk 1]
```diff
@@ -114,13 +114,47 @@
...
 
+
+      var websocketChat = {
+        send: function (message) {
+          rtc._socket.send(message);
+        },
+        recv: function (message) {
+          return message;
+        },
+        event: 'receive_chat_msg',
```
Comment (sarenji) [inline example/index.html]: I'm not sure about these commas ending the hash. I would definitely prefer it normally, but I recall that they're not syntactically valid JS.

Comment (jbenet) [inline example/index.html]: Addressed in af0231f. 

FWIW, i think the whole codebase is pretty messy and could use cleanup + style fixing. I can help with this later on, as I will be using webrtc.io for another project.

**Comment by sarenji** [comment]
Forgot to merge this as well. FWIW, I also agree regarding style.

**Pull request merged**